docs/UserGuide.md

```diff
@@ -11,8 +11,13 @@ title: User Guide
 --------------------------------------------------------------------------------------------------------------------
 ## 1. Introduction to WorkBook
 
+Are you a Computing student that finds it really hard to collate and manage all your internship progress together?
+Do you find it really hard to get an internship because you do not know how to prepare for the different interview stages? Fret not,
+WorkBook is just the right tool for you! <br><br>
 Welcome to WorkBook! :wave: <br><br>WorkBook is an **internship application tracker** that helps Computing students **prepare sufficiently** for their upcoming interviews to **secure** that internship.
```

Comment:
![Screenshot 2022-11-01 at 2 15 42 PM](https://user-images.githubusercontent.com/77665182/199170881-26061d5b-5d07-44b7-9c54-2a941b8b69ed.png)

would it be more readable if we insert a line break before "Fret not..."?
and also after adding this, the placement of "Welcome to WorkBook!..." is kind of awkward, maybe could move it to the top, or remove it?